index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>Gabriel Santo - Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header mb--md">
+        <span class="header__logo">GS</span>
+        <button class="icon-btn" type="button">
+          <svg class="icon--md">
+            <use xlink:href="img/sprite.svg#menu-btn"></use>
+          </svg>
+        </button>
```

Comment:
- [ ] Kindly wrap the button with the `nav` tag for semantic purposes. so it will be like:
```
<nav>
  <button class="icon-btn" type="button">
      <svg class="icon--md">
        <use xlink:href="img/sprite.svg#menu-btn"></use>
      </svg>
  </button>
</nav>
```